packages/compass-query-bar/src/stores/query-bar-store.js

```diff
@@ -753,8 +755,9 @@ const configureStore = (options = {}) => {
   }
 
   if (options.namespace) {
-    store.onCollectionChanged(options.namespace);
+    store.onCollectionChanged(options.namespace, options.isTimeSeries);
```

Comment:
Where is this coming from?

Reply:
It is already in the store and is being set when a new collection is selected.